README.md

```diff
@@ -19,9 +29,9 @@ https://samtecspg.github.io/articulate/
 
 3. Unzip and get a terminal or CMD in the release directory
 
-4. run `docker-compose up` (`sudo` might be needed)
+4. run `docker-compose up` (`sudo` might be needed, you can use -d after the up for it to be a daemon and run in the background)
```

Comment:
@milutz should we just do the `-d` by default, with no explanation? I want the instructions to look simple. Not have a bunch of "gotchas" tagged on.